loopd/swapclient_server.go

```diff
@@ -34,6 +35,17 @@ const (
 	minConfTarget = 2
 )
 
+var (
+	// errIncorrectChain is returned when the format of the
+	// destination address provided does not match the active chain.
+	errIncorrectChain = errors.New("invalid address format for the " +
```

Comment:
errIncorrectChain vs errInvalidAddrFormat ... ?

Reply:
`errIncorrectChain` works for me 👍